.codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
+  - '/home/travis/build/::'
```

Comment:
This is necessary since the `lcov.info` file contains absolute paths we have to rewrite.